src/classification.jl

```diff
@@ -107,13 +107,13 @@ immutable LabelMap{K}
     vs::Vector{K}
     v2i::Dict{K,Int}
 
-    function LabelMap(vs, v2i)
+    function LabelMap{K}(vs, v2i) where K
```

Comment:
Sure, that'd be great, thanks. After that you should be able to drop the dependency on Compat altogether.